VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -291,7 +298,9 @@ public void onEventMainThread(DiscussionThreadPostedEvent event) {
             // move the ListView's scroll to that newly added post's position
             discussionPostsListView.setSelection(i);
             // In case this is the first addition, we need to hide the no-item-view
-            setNoResultView(null);
+            setScreenStateUponResult(null);
```

Comment:
Or we could just add a dummy item `NONE` in the `EmptyQueryResultsFor` enum and pass it here.